jira/util/reports_generation/scale_profile.yml

```diff
@@ -1,7 +1,7 @@
 # Defines which column from test runs is used for aggregated report. Default is "90% Line"
 column_name: "90% Line"
 runs:
-  # fullPath should contain full path to directory with results.csv. E.g. /home/$USER/centaurus/jira/results/2019-08-06_17-41-08
+  # fullPath should contain full path to results.csv. E.g. /home/$USER/centaurus/jira/results/2019-08-06_17-41-08
```

Comment:
fullPath should contain a full path to the directory with run resutls. E.g. /home/$USER/dc-app-performance-toolkit/jira/results/2019-08-06_18-41-08

Reply:
done